easybuild/easyconfigs/g/GTK2/GTK2-2.24.33-GCCcore-11.3.0.eb

```
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake'
+
+name = 'GTK2'
+version = '2.24.33'
+
+homepage = 'https://developer.gnome.org/gtk+/stable/'
```

This page is dead, lets switch it to just
```suggestion
homepage = 'https://www.gtk.org'
```
for a lack of a better alternative.